container-search/src/main/resources/configdefinitions/search.query.profile.config.query-profiles.def

```diff
@@ -52,10 +52,14 @@ queryprofile[].queryprofilevariant[].inherit[] string
 queryprofile[].queryprofilevariant[].property[].name string
 # Content of profile variant
 queryprofile[].queryprofilevariant[].property[].value string
+# Whether the is overridable: "true", "false" or ""
```

Comment:
"whether the" what? (missing, i would guess "property") "is overridable"

Reply:
Fixed, thanks.